.github/workflows/release.yml

```
@@ -207,6 +221,6 @@ jobs:
             -a ./swagger.json \
             -a ./docker-stub.zip \
             -m "Version $RELEASE_VERSION" \
```

This will re-use the existing title and body.

```suggestion
            -m "" \
```